README.md

```diff
@@ -42,6 +42,38 @@ OTBR also provides the following services:
 
 Third-party components for Border Router Services include Simple Web Server and Material Design Lite for the framework of the web UI.
 
+# Get started
+
+We follow philosophy of [Scripts to Rule Them All](https://github.com/github/scripts-to-rule-them-all). Scripts are tested under,
+
+* Raspberry PI2
```

Comment:
this should be "Raspberry Pi 3B (running [Raspbian Jessie Lite)](https://www.raspberrypi.org/downloads/raspbian/)"